jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -9,14 +9,33 @@
 from jdaviz.core.events import AddDataMessage, SliceToolStateMessage, SliceSelectSliceMessage
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin
+from jdaviz.core.user_api import PluginUserApi
 
 __all__ = ['Slice']
 
 
 @tray_registry('cubeviz-slice', label="Slice")
 class Slice(PluginTemplateMixin):
+    """
+    See the :ref:`Slice Plugin Documentation <slice>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``slice``
+      Current slice number.
+    * ``wavelength``
+      Wavelength of the current slice.  If setting, will update automatically to the nearest
+      slice.
+    * ``setting_show_indicator``
```

Comment:
Now that this is public, should we remove the redundant "setting" prefix?
```suggestion
    * ``show_indicator``
```

Reply:
I'm not sure.  I have been using this prefix across all plugins for any switch that goes in the settings expansion panels of that plugin.  But we can remove the prefix from all of them, or could invent a `plugin.settings` container to nest them?

Reply:
Is that a lot of extra work? It is just a little confusing for me to see some with "settings" and some not. To me, they are all settings. I don't think API-wise, the user cares if it is nested in Settings panel or not.

Reply:
I like having some distinction between switches that only affect visualization or how many details are shown, verse inputs that actually affect the output of the plugin.  But maybe just ordering them in the API docs could accomplish that without the prefix or nesting.  I'll give it some thought.

Reply:
I support dropping the `setting_` prefix, but don't feel too strongly about it.

Reply:
ok, that would affect slice, spectral extraction, and plot options... none of which have been merged yet, so I'll update the other 2 open PRs separately as well.